tst/com/amazon/ion/benchmark/OptionsTest.java

```diff
@@ -2102,4 +2107,128 @@ public void testDetectRegression() {
         double realResult = ParseAndCompareBenchmarkResults.detectRegression(before, after);
         assertEquals(expectResult, realResult, 1e-16);
     }
+
+    @Test
+    public void readIon11() throws Exception {
+        ReadOptionsCombination optionsCombination = parseSingleOptionsCombination(
+            "read",
+            "--ion-minor-version",
+            "1",
+            "--io-type",
+            "buffer",
+            "binaryAllTypes11.10n"
+        );
+        assertReadTaskExecutesCorrectly(
+            "binaryAllTypes11.10n",
+            optionsCombination,
+            Format.ION_BINARY,
+            false
+        );
+    }
+
+    @Test
+    public void readIon11WithConversionFrom10() throws Exception {
```

Comment:
I'm wondering if the condition would happen when the input data is 1.1 encoded but the minor version number is specified as 0.

Reply:
Yes, it will first convert the 1.1 data to 1.0 and then read. I've added testing of this case in revision 2 of the PR.